lms/static/js/i18n/en@lolcat/djangojs.js

```diff
@@ -19,42 +19,6 @@
 
   django.catalog = django.catalog || {};
   
-  var newcatalog = {
-    "%(sel)s of %(cnt)s selected": [
-      "%(sel)s of %(cnt)s selected",
-      "%(sel)s of %(cnt)s selected"
-    ],
-    "6 a.m.": "6 a.m.",
-    "Available %s": "Available %s",
-    "Cancel": "Cancel",
-    "Choose": "Choose",
-    "Choose a time": "Choose a time",
-    "Choose all": "Choose all",
-    "Chosen %s": "Chosen %s",
-    "Click to choose all %s at once.": "Click to choose all %s at once.",
-    "Click to remove all chosen %s at once.": "Click to remove all chosen %s at once.",
-    "Filter": "Filter",
-    "Hide": "Hide",
-    "Midnight": "Midnight",
-    "Noon": "Noon",
-    "Now": "Now",
-    "Remove": "Remove",
-    "Remove all": "Remove all",
```

Comment:
@attiyaIshaque I think this block is mistakenly removed. Please add this back as this is a new change in the Juniper release. Also, check for any other translations mistakenly removed that should be part of the Juniper release